content/posters/2024/Q4/Group 24/students/Nils Mikk/student_info.yaml

```
@@ -0,0 +1,5 @@
+thesisTitle: 'Program Synthesis from Rewards with Probe: Adjusting Probe to Increase Exploration when Synthesizing Programs From Rewards in Minecraft'
```

Change `z` to `s` in `Synthesizing`, and `F` to lowercase in `From`.
```suggestion
thesisTitle: 'Program Synthesis from Rewards with Probe: Adjusting Probe to Increase Exploration when Synthesising Programs from Rewards in Minecraft'
```